src/Eurofurence.App.Server.Services/Events/EventService.cs

```diff
@@ -28,5 +57,232 @@ public IQueryable<EventRecord> FindConflicts(DateTime conflictStartTime, DateTim
                 e.StartDateTimeUtc <= queryConflictEndTime &&
                 e.EndDateTimeUtc >= queryConflictStartTime);
         }
+
+        public async Task RunImportAsync()
+        {
+            _logger.LogInformation(LogEvents.Import, "Starting events import.");
+
+            var httpClient = new HttpClient();
+            var fileStream = await httpClient.GetStreamAsync(_configuration.Url);
+            TextReader reader = new StreamReader(fileStream);
+
+            using var csv = new CsvReader(reader, new CsvConfiguration(CultureInfo.CurrentCulture) { Delimiter = "," });
+            csv.Context.RegisterClassMap<EventImportRowClassMap>();
+            var csvRecords = csv.GetRecords<EventImportRow>().ToList();
+            csvRecords = csvRecords
+                .Where(a => !a.Abstract.Equals("[CANCELLED]", StringComparison.CurrentCultureIgnoreCase)).ToList();
+
+            if (csvRecords.Count == 0) return;
+
+            foreach (var record in csvRecords)
+                record.ConferenceDayName = record.ConferenceDayName.Contains(" - ")
+                    ? record.ConferenceDayName.Split(new[] { " - " }, StringSplitOptions.None)[1].Trim()
+                    : record.ConferenceDayName.Trim();
+
+            var conferenceTracks = csvRecords.Select(a => a.ConferenceTrack)
+                .Distinct().OrderBy(a => a).ToList();
+
+            var conferenceRooms = csvRecords.Select(a => a.ConferenceRoom)
+                .Distinct().OrderBy(a => a).ToList();
+
+            var conferenceDays = csvRecords.Select(a =>
+                    new Tuple<DateTime, string>(DateTime.SpecifyKind(DateTime.Parse(a.ConferenceDay), DateTimeKind.Utc),
```

Comment:
Similar case here, `DateTime.Parse` uses the current culture, so if you are running on a German computer it parses `dd.MM.yyyy HH:mm:ss` and on a US computer `MM/dd/yyyy hh:mm:ss`. Since the format is not going to change, using `DateTime.ParseExact` with a format string should work. (if it's an ISO 8601 date, you can just use `o` as the format string)